### DISCUSSION TITLE
Regarding Values returned using Centrality Algorithms

### DISCUSSION THREAD

**Discussion by 0ddoes** [networkx/networkx#5541] [Q&A] [answered]
Why are the values that are returned using Centrality Algorithms not rounded to the nearest 3-4 decimal places. Is there some particular reason for it because I feel interpretation would be neater if the values were rounded off?

**Comment by MridulS** [comment, marked as answer]
Hmm, I don't think we should be controlling and stripping the float values and sometimes it's absolutely makes sense to return the long decimal value. If the user wants they can easily change the format of the decimal values in their analysis. We should let python handle this.
Checkout https://docs.python.org/3/tutorial/floatingpoint.html if you are interested in reading more about how floats work in python :)
